vars.tf

```diff
@@ -18,3 +18,9 @@ variable "lock" {
   description = "Lock ressource group for deletion"
   default     = true
 }
+
+variable "proximiy_placement_group" {
```

Comment:
There's a typo here. A t is missing. And I'd suggest using the parameter "create_proximity_placement_group" or "manage_"